src/neptune/internal/operation_processors/factory.py

```diff
@@ -54,6 +57,8 @@ def get_operation_processor(
     async_no_progress_threshold: float = ASYNC_NO_PROGRESS_THRESHOLD,
 ) -> OperationProcessor:
     if mode == Mode.ASYNC:
+        batch_size = int(os.environ.get(NEPTUNE_ASYNC_BATCH_SIZE) or "1000")
```

Comment:
Should we assert input value? :)

Reply:
True. Is there an upper limit for a batch size?

Reply:
Not really. 100k may work for production but this could be customized.